hassio/addon-view/hassio-addon-audio.html

```diff
@@ -0,0 +1,119 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../bower_components/paper-card/paper-card.html">
+<link rel="import" href="../../bower_components/paper-button/paper-button.html">
+<link rel="import" href="../../bower_components/paper-dropdown-menu/paper-dropdown-menu.html">
+<link rel='import' href='../../bower_components/paper-listbox/paper-listbox.html'>
+<link rel='import' href='../../bower_components/paper-item/paper-item.html'>
+<link rel='import' href='../../bower_components/neon-animation/web-animations.html'>
```

Comment:
You're not using this?

Reply:
web-animations? I need the for the paper-listbox

Reply:
Doesn't paper-dropdown-menu import that via paper-menu-button?
I don't see a direct use in this component, only secondary.

Reply:
without you get this error and the dropdown doesn't close
```
Uncaught ReferenceError: KeyframeEffect is not defined
    at HTMLElement.configure (fade-in-animation.html:39)
    at HTMLElement._configureAnimations (neon-animation-runner-behavior.html:44)
    at HTMLElement.playAnimation (neon-animation-runner-behavior.html:114)
    at HTMLElement._renderOpened (iron-dropdown.html:214)
    at HTMLElement.__openedChanged (iron-overlay-behavior.html:597)
    at HTMLElement.nextAnimationFrame (iron-overlay-behavior.html:620)
```